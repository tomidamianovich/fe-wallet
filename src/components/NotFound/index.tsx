import React from 'react';
import './styles/index.scss';

type Props = {};

const NotFound: React.FC<Props> = () => 
  <header className="fe-wallet__not-found">
    Not Found - 404
  </header>
;

export default NotFound;