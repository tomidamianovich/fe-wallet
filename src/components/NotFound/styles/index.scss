@import "../../../containers/App/styles/colours.scss";

.fe-wallet {

  
}