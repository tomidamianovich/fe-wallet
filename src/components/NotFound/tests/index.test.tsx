import { render } from '@testing-library/react';
import NotFound from '../';

jest.mock('react-i18next', () => ({
  withTranslation: () => (Component: any) => {
    Component.defaultProps = {...Component.defaultProps, t: (translation: string) => translation};
    return Component;
  }
}));


test('renders Exchange main page', () => {
  const { asFragment } = render(<NotFound />);
  expect(asFragment()).toMatchSnapshot();
});