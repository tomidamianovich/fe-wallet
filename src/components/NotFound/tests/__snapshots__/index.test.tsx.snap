// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Exchange main page 1`] = `
<DocumentFragment>
  <header
    class="fe-wallet__not-found"
  >
    Page Not Found
  </header>
</DocumentFragment>
`;
